apple/Tests/FerrostarSwiftUITests/Theme/ArrivalViewThemeTests.swift

```diff
@@ -0,0 +1,15 @@
+import XCTest
+@testable import FerrostarSwiftUI
+
+final class ArrivalViewThemeTests: XCTestCase {
```

Comment:
I doubt we need this test case TBH; it'll just be one more thing to change if we change our mind on the default constructor args 😂

Reply:
Agree this test is generally useless. The only two reasons I added it anyways are:

1. It's a good additional catch for misconfigurations that could lead to somewhat larger issues (e.g. missing text)
2. It matches what we did for the instructions view. 

We could also just snapshot test the full default ferrostar theme once we've got all the ui built.

Reply:
> It matches what we did for the instructions view.

😂 I guess feel free to delete that then; I must have missed it.

I'd say visual snapshot image tests of the UI are probably the way to go in issue detection here. Unless we have a specific edge case that we're trying to guard against. And that's probably solvable with other techniques that make illegal state unrepresentable.

Reply:
It's gone! 😆